grisera_api/registered_channel/registered_channel_service_relational.py

```diff
@@ -1,5 +1,73 @@
+        import registered_data.registered_data_service_relational
+        registered_data_service_relational = registered_data.registered_data_service_relational.RegisteredDataServiceRelational()
+        if depth > 0:
+            registered_channel["registeredData"] = registered_data_service_relational.get_registered_data(
+                registered_channel["registered_data_id"], depth - 1, self.table_name)
+            # registered_channel["channels"] = self.channels_service.get_channel(
+            #TODO     registered_channel["channel_id"], depth - 1, self.table_name)
+            # if source != Collections.RECORDING:
+            #     registered_channel["recordings"] = self.recording_service.get_recording(
+            #         registered_channel["registered_data_id"], depth - 1, self.table_name)
+        return RegisteredChannelOut(**registered_channel)
+
+    def get_multiple_with_foreign_id(self, foreign_id: Union[int, str], depth: int = 0, source: str = ""):
+        response = self.rdb_api_service.get_records_with_foreign_id(self.table_name, source+"_id",foreign_id)
+        if "errors" in response.keys():
+            print(response["errors"])
+            return []
+        registered_channels = response["records"]
+        import registered_data.registered_data_service_relational
+        registered_data_service_relation = registered_data.registered_data_service_relational.RegisteredDataServiceRelational()
+        if depth > 0:
+            for registered_channel in registered_channels:
+                if source != Collections.REGISTERED_DATA:
+                    registered_channel["registeredData"] = registered_data_service_relation.get_registered_data(
+                        registered_channel["registered_data_id"], depth - 1, self.table_name)
+                # if source != Collections.CHANNEL:
+                #TODO     registered_channel["channels"] = self.channels_service.get_channel(
+                #         registered_channel["channel_id"], depth - 1, self.table_name)
+                # registered_channel["recordings"] = self.recording_service.get_multiple_with_foreign_id(
+                #     registered_channel["registered_data_id"], depth - 1, self.table_name)
+        return registered_channels
+
+    def update_registered_channel_relationships(self, registered_channel_id: Union[int, str],
+                                                registered_channel: RegisteredChannelIn):
+        result = self.get_registered_channel(registered_channel_id)
+        if type(result) != NotFoundByIdModel:
+            put_result = self.rdb_api_service.put(self.table_name, registered_channel_id, registered_channel.dict())
+            if put_result["errors"] is not None:
+                return RegisteredChannelOut(errors=put_result["errors"])
+            return RegisteredChannelOut(**put_result["records"])
+        return result
```

Comment:
Jeśli record z takim ID nie istnieje powinno być zwrócone NotFoundByIdModel(id=registered_channel_id, errors="Entity not found.")

Reply:
Faktycznie, wtedy jest dobrze